v0/destinations/facebook_pixel/transform.js

```diff
@@ -178,10 +188,18 @@ const handleProductListViewed = (message, categoryToContent) => {
   }
   contents.forEach((content, index) => {
     if (content.id === "") {
-      throw new CustomError(
-        `Product id is required for product ${index}. Event not sent`,
-        400
-      );
+      if(contentType === "product") {
+        throw new CustomError(
+          `Product id is required for product ${index}. Event not sent`,
+          400
+        );
+      } else {
+        throw new CustomError(
+          "In order to send viewContent event for payload without products, category is a mandatory field. Event not sent",
+          400
+        );
+      }
```

Comment:
Can you elaborate this logic?

Reply:
The explanation you are finding is on lines 182-187